### ISSUE TITLE
ISR not working with GrovePi shield on Up2

### ISSUE THREAD

**Issue by srware** [intel-iot-devkit/mraa#888]
Up2 board + GrovePi shield, Ubuntu Server 16.04, MRAA 1.9.0 from PPA

I tried using the Javascript ISR example with a button connected through D4 on the GrovePi shield and got no response for any configuration (EDGE_RISING, EDGE_FALLING, EDGE_BOTH).

Manually polling the button value works OK.

Is this expected?

**Comment by Propanu** [comment]
Yes, ISR support wasn't added yet the GrovePi shield. See [here](https://github.com/intel-iot-devkit/mraa/blob/master/src/grovepi/grovepi.c#L276).
